### ISSUE TITLE
No quests counting after taking "Heated Battle" (Aliance)

### ISSUE THREAD

**Issue by thesteadystone** [dalaranwow/dalaran-wow#5970]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Dear Dalaran Team. After taking the quest "Heated Battle" on my character Zindera, I have not been able to gain "credits" (if this is the right word) for any other quests. Any of the quests which requires some kind of counting, "slains", gathers and so on, refuse to register. 
My friend who I quest with is having the same issue.

**Expected behaviour**: Tell us what should happen instead
I have tried taking many quests which I know to work, including daliy quests I have already tried on this character, and none of them will register "credits".  

**Steps to reproduce the problem**:

1. I tried to take some of the same quests on an other charater, and these were working.
2. I am not certain if the problem is related to the quest "Heated Battle", but I am not currently prepered for trying it out on another of my characters.
3. The issue is the exact same on two different accounts. 

**Include proofs for this behaviour**
I am not certain how I should "proof" this, but I have tried to delete the cache and WTF folder, without any difference.
I also tried to abandon all quests and then try some new ones without succes. 
I also tried to reinstall the client, although I could not see why this should work.
Furthermore it seems like the quest "Heated Battle" is not working - all the NPC are in a constant mellay, but then again, we did not get any credit for slaying the mobs, so it might just be liked to our issue.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
